src/components/Books.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = (props) => {
+  const { title, author } = props;
+  return (
+    <div>
+      <h2>{title}</h2>
+      <h3>{author}</h3>
+    </div>
+  );
```

Comment:
- [ ] Please add a button for removing books as per the requirement. It should be a placeholder for now.